ruby/vampires.rb

```diff
@@ -0,0 +1,74 @@
+puts "How many employees will be processed?"
+employees = gets.chomp.to_i
+
+employees.times do |x|
+  puts "What is your name?"
+  username = gets.chomp 
+  
+  puts "How old are you?"
+  age = gets.chomp.to_i
+  
+  puts "What year were you born?"
+  birth_year = gets.chomp.to_i
+  
+  actual_age = Time.now.year - birth_year
+  correct_age = actual_age == age
+  
+  puts "Our company cafeteria serves garlic bread. Should we order some for you?"
+  garlic = gets.chomp
+  
+  puts "Would you like to enroll in the company’s health insurance?"
+  health = gets.chomp 
+  
+  my_allergy = nil
```

Comment:
If I'm conditionally doing a loop based on user input, I'd probably just use a `loop do` iterator and then just have a break statement for when I want to get out of it.  You may want to consider having this loop set a variable that is leveraged in the if blocks.